slice.go

```diff
@@ -182,7 +182,7 @@ func Chunk[T any, Slice ~[]T](collection Slice, size int) []Slice {
 		if last > len(collection) {
 			last = len(collection)
 		}
-		result = append(result, collection[i*size:last])
+		result = append(result, collection[i*size:last:last])
```

Comment:
I had never faced this notation, with the capacity apparently

Reply:
ref: https://stackoverflow.com/questions/12768744/re-slicing-slices-in-golang/18911267#18911267